VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -113,26 +113,26 @@ final internal class ResponseCache
     
     private class ResponseMemoryCache: Cache
     {
-        private let cache = NSCache()
+        private let cache = NSCache<AnyObject, AnyObject>()
```

Comment:
NSCache defines its two types, `KeyType` and `ObjectType` as `AnyObject`:

```
open class NSCache<KeyType : AnyObject, ObjectType : AnyObject>
```

This was the thing I was referring to this morning, and I'm not sure there's any way around it. If you have suggestions or anything let me know.